### ISSUE TITLE
xCAT Probe sub-command switch_macmap miss dependency in ubuntu

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#6313]
xCAT Probe sub-command ``switch_macmap`` miss dependency in ubuntu.

```
root@c910f04x12v05:/opt/xcat/probe/subcmds# lsxcatd -a
Version 2.15 (git commit 7873494d3c00a9232745990bee2f2f442198d4ef, built Tue May 14 06:21:19 EDT 2019)
This is a Management Node
dbengine=SQLite

root@c910f04x12v05:/opt/xcat/probe/subcmds# cat /etc/*release*
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.5 LTS"
NAME="Ubuntu"
VERSION="16.04.5 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.5 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial

root@c910f04x12v05:/opt/xcat/probe/subcmds# pwd
/opt/xcat/probe/subcmds

root@c910f04x12v05:/opt/xcat/probe/subcmds#  ./switch_macmap c910f04x12v06
[debug]  :sh: /opt/xcat/probe/subcmds/bin/switchprobe: No such file or directory
``` 

In xCAT RPM version, this setting is configured in ``xCAT-probe.spec``.
```
.......
%post
if [ -e %{prefix}/probe/subcmds/bin/switchprobe ]; then
    rm -rf %{prefix}/probe/subcmds/bin/switchprobe
else
    mkdir -p %{prefix}/probe/subcmds/bin/
fi
cd %{prefix}/probe/subcmds/bin/
if [ -e %{prefix}/bin/xcatclient ]; then
    ln -s %{prefix}/bin/xcatclient switchprobe
fi
```

But miss this in xCAT ubuntu build script ``...xcat-core/xCAT-probe/debian/rules`` .

**Comment by gurevichmark** [comment]
Resolved by #6320

**Comment by gurevichmark** [comment]
Fixed
